src/utils/helper.ts

```diff
@@ -26,12 +24,8 @@ export const categories = {
     label: "Packaging Recyclability",
     colour: Colours.Skobeloff,
   },
-  disclosure: { label: "Animal Testing", colour: Colours.SkyBlueCrayola },
-  humanHealth: {
-    label: "Human Health",
-    colour: Colours.NaplesYellow,
-  },
-  supplyChainAndEnvironment: {
+  animalTesting: { label: "Animal Testing", colour: Colours.SkyBlueCrayola },
+  manufacturingEmissions: {
     label: "Manufacturing greenhouse gas emissions",
```

Comment:
Sorry I missed this earlier 
```suggestion
    label: "Manufacturing Emissions",
```